variables.tf

```diff
@@ -48,3 +48,13 @@ variable ssh_key {
     default     = ""
     description = "Your ssh key, examples: 'github: myghid' or 'ssh-rsa AAAAblahblah== keyname'" 
 }
+
+variable num_of_vlans {
+  default = 0
+  description = "Number of VLANs to be created"
+}
+
+variable network_type {
```

Comment:
I don't see this variable being used anywhere. Looks like `hybrid` is hardcoded in main.tf.

Reply:
Missed it. Will update